packages/manager/modules/manager-components/src/region/service.js

```diff
@@ -23,28 +24,30 @@ export default class CucRegionService {
 
   getAllTranslatedMacroRegion() {
```

Comment:
There is a requirement to show region with city name, country name and datacenter name like "Strasbourg (SBG1) France". Is it supported?

Reply:
@radireddy Not sure I've just kept it as is to not break existing usage. I guess if we have other need we can extend the service to add needed formatters/functions